### PR TITLE
fix warnings introduced with the newlayout merge

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: build/FindQt5Transitional.cmake [hunk 1]
```diff
@@ -23,8 +23,8 @@ if (Qt5Core_FOUND)
         OpenGL
         LinguistTools
         Help
-        WebEngine
-        WebEngineWidgets
+#        WebEngine
+#        WebEngineWidgets
```
Comment (Jojo-Schmitz) [inline build/FindQt5Transitional.cmake]: There may be a better way to do this, so it only affects MINGW?

File: mscore/musescore.cpp [hunk 2]
```diff
@@ -2302,17 +2302,22 @@ static void mscoreMessageHandler(QtMsgType type, const QMessageLogContext &conte
      QByteArray localMsg = msg.toLocal8Bit();
 
      switch (type) {
+ #if (QT_VERSION >= QT_VERSION_CHECK(5, 5, 0))
```
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: more of a documentation than really needed

File: build/FindQt5Transitional.cmake (hunk 1 above)
Comment (Jojo-Schmitz) [inline build/FindQt5Transitional.cmake]: Thwere got to be, as this way it doesn't work. The cmake warningis are annoying though

File: mscore/musescore.cpp (hunk 2 above)
Comment (lasconic) [inline mscore/musescore.cpp]: No way. If we put this every single time we use qInfo... the code will get cluttered. So or we swallow the pill or we don't, there is no in the middle solution.

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: OK, fixed

File: mscore/exportxml.cpp [hunk 3]
```diff
@@ -5131,6 +5131,8 @@ void ExportMusicXml::write(QIODevice* dev)
                                     xml.tag("staff-distance", QString("%1").arg(QString::number(staffDist,'f',2)));
                                     xml.etag();
                                     }
+                              }
+                        }
 #endif //MERGE
```
Comment (Jojo-Schmitz) [inline mscore/exportxml.cpp]: These unbalanced braces, regardless the code section being disabled, causes a warning from lupdate.

File: mscore/musescore.cpp [hunk 4]
```diff
@@ -2302,17 +2302,20 @@ static void mscoreMessageHandler(QtMsgType type, const QMessageLogContext &conte
      QByteArray localMsg = msg.toLocal8Bit();
 
      switch (type) {
+     case QtInfoMsg:
+         cerr << "Info: "     << localMsg.constData() << " ("  << context.file << ":" << context.line << ", " << context.function << ")" << endl;
+         break;
      case QtDebugMsg:
-         cerr << "Debug: " << localMsg.constData() << " ("  << context.file << ":" << context.line << ", " << context.function << ")" << endl;
+         cerr << "Debug: "    << localMsg.constData() << " ("  << context.file << ":" << context.line << ", " << context.function << ")" << endl;
```
Comment (lasconic) [inline mscore/musescore.cpp]: What's the problem with the spaces here?

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: no real problem other than alignment, that way it is easier to spot that all the lines are identical, except for the prefix (Info:, Debug:, Critical:, Fatal:)

Comment (lasconic) [inline mscore/musescore.cpp]: That's definitely the kind of things that I would be happy to loose if we can agree on a tool to format the code.

Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: maybe we could factor out that common part?

**Pull request merged**